src/main/java/step3/domain/LottoTicket.java

```diff
@@ -0,0 +1,59 @@
+package step3.domain;
+
+import static step3.constant.LottoInfoConstant.LOTTO_DELIMITER;
+import static step3.constant.LottoInfoConstant.LOTTO_ELEMENTS_SIZE;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoTicket {
+
+    private final ArrayList<LottoElement> lottoElements = new ArrayList();
```

Comment:
다형성을 위해서 다음과 같이 수정하면 좋을 것 같습니다.
```suggestion
    private final List<LottoElement> lottoElements = new ArrayList();
```